data-model/pom.xml

```diff
@@ -58,6 +58,11 @@
             <artifactId>utils</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This is added because in `data-model` we need annotations, while the API definition is done in `rest-api` module.